lib/rules/no-setup-props-reactivity-loss.js

```diff
@@ -139,15 +139,26 @@ module.exports = {
       while (expression.type === 'MemberExpression') {
         expression = utils.skipChainExpression(expression.object)
       }
-      /** A list of expression nodes to verify */
-      const expressions =
-        expression.type === 'TemplateLiteral'
-          ? expression.expressions
-          : expression.type === 'ConditionalExpression'
-            ? [expression.test, expression.consequent, expression.alternate]
-            : expression.type === 'Identifier'
-              ? [expression]
-              : []
+      /** @type {Expression[]} A list of expression nodes to verify */
+      const expressions = []
+      switch (expression.type) {
+        case 'TemplateLiteral': {
+          expressions.push(...expression.expressions)
+          break
+        }
+        case 'ConditionalExpression': {
+          expressions.push(
+            expression.test,
+            expression.consequent,
+            expression.alternate
+          )
+          break
+        }
+        case 'Identifier': {
+          expressions.push(expression)
+          break
+        }
+      }
```

Comment:
This change is unrelated, right? It looks like a good change, so no need to revert it or anything, I just stumbled a bit over it.